QRCodeBasedMetroTicketingSystem/QRCodeBasedMetroTicketingSystem.Web/wwwroot/js/rapidpass-and-qr-ticket-handler.js

```
@@ -0,0 +1,159 @@
+        }
+
+        // Initial call
+        updateTimer();
+
+        // Start countdown interval
+        window.countdownInterval = setInterval(updateTimer, 1000);
+    }
+
+    // Download QR Ticket and RapidPass QR code as PDF (when clicking the download button on the ticket card or clicking the download button in the modal)
+    $('.download-ticket').on('click', function () {
+        var btn = $(this);
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)